docker-compose.yaml

```diff
@@ -1,12 +1,27 @@
 version: "3"
 services:
-  node_dev_env:
+  app:
     restart: always
     build: .
     container_name: wolfsbane-backend
+    links:
+      - postgres
+    environment:
+      DEV_DATABASE_URL: postgres://devuser:1234@postgres:5432/dev_db
+      NODE_ENV: development
+      PORT: 3000
     ports:
       - 3000:3000
     command: npm run start:dev
     volumes:
       - .:/app/
       - /app/node_modules
+
+  postgres:
+    image: postgres:10.4
```

Comment:
Any specific reason for going with version 10.4 of postgres and not the latest

Reply:
Ok @sa-ma , No much specific reason, but rather based on best practices, we need to be version-specific, doing this prevents the app from breaking whenever a new version different from the version used while building the app is released. At the long run, when new versions are released some functions on which our app depends on might become obsolete leading to the breakdown of the app

Reply:
No problem, rebase so I can merge you have conflicts

Reply:
Done